ow_lander/scripts/arm_action_servers.py

```diff
@@ -793,6 +796,86 @@ def now_from_start(start):
             self._server.set_aborted(self._result)
 
 
+class ARM_MOVE_JOINT(object):
+
+    def __init__(self, name):
+        self._action_name = name
+        # Action Feedback/Result
+        self._fdbk = ow_lander.msg.ARM_MOVE_JOINTFeedback()
+        self._result = ow_lander.msg.ARM_MOVE_JOINTResult()
+        self._current_joint_state = JointStateSubscriber()
+        self._interface = MoveItInterface()
+        self._timeout = 0.0
+        self._server = actionlib.SimpleActionServer(self._action_name,
+                                                    ow_lander.msg.ARM_MOVE_JOINTAction,
+                                                    execute_cb=self.on_arm_move_joint_action,
+                                                    auto_start=False)
+        self._server.start()
+
+    def _update_feedback(self):
+        self._js = self._current_joint_state.get_joint_angles()
+        self._fdbk.angle = self._js
+        self._server.publish_feedback(self._fdbk)
+
+    def _update_motion(self, goal):
+
+        rospy.loginfo("Arm move joint started")
+        joint_goal = self._interface.move_arm.get_current_joint_values()
+
+        if goal.relative == False:
+            joint_goal[goal.joint] = goal.angle
+        else:
+            joint_goal[goal.joint] = joint_goal[goal.joint] + goal.angle
```

Comment:
[optional] I think this reads better
```suggestion
        if goal.relative:
            joint_goal[goal.joint] = joint_goal[goal.joint] + goal.angle
        else:
            joint_goal[goal.joint] = goal.angle
```